project/Component/NefModels/ProgressReport.swift

```diff
@@ -37,6 +37,13 @@ public enum ProgressEventStatus {
 
 public protocol CustomProgressDescription {
     var progressDescription: String { get }
+    var currentStep: UInt { get }
+    var totalSteps: UInt { get }
+}
+
+public extension CustomProgressDescription {
+    var currentStep: UInt { 0 }
+    var totalSteps: UInt { 0 }
```

Comment:
I think `1` is a more sensible default value in both cases. It prevents division by zero errors if someone attempts to render some kind of percentage, without needing to check if it is zero or not.